app/helpers/admin/templates_helper.rb

```diff
@@ -10,7 +10,7 @@ def example_data
       selection_mail: {
         applicant_name: "Ruby",
         event_date: @event.scheduled_at.strftime("%d.%m.%Y"),
-        confirmation_deadline: (@event.confirmation_date + 5.days).strftime("%d.%m.%Y"),
+        confirmation_deadline: (@event.confirmation_date + @event.confirmation_deadline).strftime("%d.%m.%Y"),
```

Comment:
Seems we are repeating the same `strftime("%d.%m.%Y")` part of the code in multiple places. 
Maybe we could avoid the repetition and try ensure that the dates are always with the same format? Moving this to a helper could be an option.

Reply:
To be done in a separate PR :)